src/components/dialog-full-screen/dialog-full-screen.stories.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import notes from './notes.md';
+import { State, Store } from '@sambego/storybook-state';
+import Dialog from './dialog-full-screen';
+import Button from '../button';
+
+const store = new Store({
+  open: false
+});
+
+const handleCancel = () => {
+  store.set({ open: false });
+  action('cancel')();
+};
+const handleOpen = () => {
+  store.set({ open: true });
+  action('open')();
+};
+
+storiesOf('Dialog Full Screen', module)
+  .add('default', () => {
+    const open = boolean('open', store.open);
+    const title = text('title', 'Example Dialog');
+    const subtitle = text('subtitle', 'Example Subtitle');
+    const children = text('children', 'Text Content');
+    const enableBackgroundUI = boolean('enableBackgroundUI', false);
```

Comment:
Can you provide default values from component?
`Dialog.defaultProps.enableBacjgroundUI`

Reply:
👍  Great idea.